jobs/JGFS_ATMOS_GEMPAK_PGRB2_SPEC

```diff
@@ -38,22 +36,18 @@ export DBN_ALERT_TYPE=GFS_GOESSIM_GEMPAK
 export RUN2=gfs_goessim
 export GRIB=goessimpgrb2.0p25.f
 export EXT=""
-export fend=180
-if (( fend > FHMAX_GFS )); then
-  fend=${FHMAX_GFS}
-fi
-export finc=3
-export fstart=0
-
-echo "RUNS the Program"
 
-########################################################
-# Execute the script.
-"${SCRgfs}/exgfs_atmos_goes_nawips.sh"
+fhmin=0
```

Comment:
The `if`s in this script can also be removed.

Reply:
`if` statements are removed.